### PR TITLE
delete workloads

### PR DESCRIPTION
**Pull request by juozasg**
Add 'Delete' right-click context menu for Kustomization and HelmRelease workloads

### CODE REVIEW

**Review by kingdonb — APPROVED** [review]
👍 I think this LGTM

File: src/azure/azureTools.ts [hunk 1]
```diff
@@ -378,6 +378,22 @@ class AzureTools {
...
+		contextName: string,
+		clusterProvider: AzureClusterProvider,
+	) {
+		const deleteSourceShellResult = await this.invokeAzCommand(
+			`az k8s-configuration flux kustomization delete -n ${kustomizationName} --yes`,
+			contextName,
+			clusterProvider,
+		);
+		if (deleteSourceShellResult?.code !== 0) {
+			telemetry.sendError(TelemetryErrorEventNames.FAILED_TO_RUN_AZ_DELETE_SOURCE);
```
Comment (usernamehw) [inline src/azure/azureTools.ts:393]: Should this be a "delete az workload" telemetry event & variable name?

Comment (kingdonb) [reply src/azure/azureTools.ts:393]: You're right, this needs changes. And some new constants to be declared as well 👍

**Review by kingdonb — APPROVED** [review]
LGTM with changes – the source constants are reused resulting in confusing log output.

They should be copied and new ones made that say "Workload" when it's a Kustomization

**Pull request merged**